service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapperTest.java

```diff
@@ -51,6 +52,10 @@
         + "immunization-reason-not-given-text.json";
     private static final String INPUT_JSON_WITH_VACCINE_CODE = IMMUNIZATION_FILE_LOCATIONS
         + "immunization-vaccine-code-given.json";
+    private static final String INPUT_JSON_WITHOUT_PRACTITIONER = IMMUNIZATION_FILE_LOCATIONS
+        + "immunization-no-practitioner.json";
+    private static final String INPUT_JSON_WITHOUT_PRACTITIONER_ACTOR = IMMUNIZATION_FILE_LOCATIONS
+        + "immunization-practitioner-but-no-actor.json";
```

Comment:
What about the scenario if an `immunization` has a `practitioner` with an `actor`?

Reply:
That's covered by existing tests - e.g. immunization-all-pertinent-information.json